node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -2252,7 +2560,7 @@ mod tests {
         assert_eq!(system.run(), 0);
         let recording = recording_arc.lock().unwrap();
         assert_eq!(recording.len(), 0);
-        assert_eq!(result, GossipAcceptanceResult::Accepted);
+        assert_eq!(result, GossipAcceptanceResult::Ignored);
```

Comment:
Put a comment in here explaining that this is Standard Gossip, even though it looks like a Debut, because it's specifically handled by a `StandardGossipHandler` instead of the `GossipAcceptor` (which would identify it as a Debut), so the test is unrealistic. Also that the Gossip is ignored because Node B isn't in Node A's patch, which matters to a `StandardGossipHandler`.